core/io/dns_config.hxx

```diff
@@ -40,7 +40,7 @@ class dns_config
     [[nodiscard]] std::chrono::milliseconds timeout() const;
 
   private:
-    std::string nameserver_{ default_nameserver };
+    std::string nameserver_{};
```

Comment:
Do we need to keep the `static constexpr auto default_nameserver = "8.8.8.8";` line?  Seems like it might be better to not even hint at a potential hard-coded default value.  I think it is used in the tests, but that is easily moved to only reside w/in the test suite.

Reply:
It is a constant, let it be there tests are using it.